BDArmory/Guidances/CruiseGuidance.cs

```diff
@@ -51,6 +52,13 @@ public class CruiseGuidance : IGuidance
         private Vector3 planarDirectionToTarget;
         private Vector3 upDirection;
 
+        private float _popupCos = -1f;
+        private float _popupSin = -1f;
+
+        // Popup 1/g
+        const float invG = 1f / 10f;
```

Comment:
Why the two almost equal `invG` and `invg` values at roughly `1f / (float)PhysicsGlobals.GravitationalAcceleration`?
Also, from what I can see, they're only used together as `invG * invg`, so why not use a single constant for the combination?
Also, does this constant change if a user messes with the gravity multiplier `PhysicsGlobals.GraviticForceMultiplier`?

Reply:
invG is 1 / the maneuver g, invg is 1/g_0. The reason why I separated the two is because I was debating whether or not invG should be a config parameter. While I went with "no", I wanted to leave it there to make it easy in case we ever wanted to add that as a config parameter.